ClientApp/src/components/pages/GoalDetailPage/UpdatesWidget.tsx

```diff
@@ -195,52 +197,55 @@ export const UpdatesWidget: React.FC<UpdatesWidgetProps> = ({
           ) : (
             <></>
           )}
-          {updates.length !== 0 ? (
+          {hasUpdates ? (
             <Accordion className="list-group-flush">
-              {updates
-                .sort((a: GoalUpdateSnippet, b: GoalUpdateSnippet): number => {
-                  return (
-                    moment(b.updateDate).valueOf() -
-                    moment(a.updateDate).valueOf()
-                  );
-                })
-                .map(
-                  (update: GoalUpdateSnippet): JSX.Element => {
-                    return (
-                      <>
-                        <ContextAwareAccordionToggle
-                          key={update.goalUpdateId}
-                          callback={handleToggle}
-                          eventKey={String(update.goalUpdateId)}
-                        >
-                          <Row>
-                            <Col>{update.title}</Col>
-                            <Col
-                              className="d-flex justify-content-between"
-                              xs={4}
-                            >
-                              {moment(update.updateDate).format("MM/DD/YYYY")}
-                              {+openToggle === update.goalUpdateId ? (
-                                <ArrowDropUpIcon />
-                              ) : (
-                                <ArrowDropDownIcon />
-                              )}
-                            </Col>
-                          </Row>
-                        </ContextAwareAccordionToggle>
-                        <Accordion.Collapse
-                          eventKey={String(update.goalUpdateId)}
-                        >
-                          <Card.Body className="sub-table">
-                            <span>
-                              {update.description || <em>No Notes</em>}
-                            </span>
-                          </Card.Body>
-                        </Accordion.Collapse>
-                      </>
-                    );
-                  },
-                )}
+              {updates ||
+                []
```

Comment:
```suggestion
              {(updates || [])
```
To fix runtime error

Reply:
Good call. Thanks